src/api/application/application.ts

```diff
@@ -175,6 +175,19 @@ export default class ApplicationModule extends Base {
         return app;
     }
 
+    /**
+     * Asynchronously starts a batch of applications given an array of application identifiers and manifestUrls.
+     * Returns once the RVM receives the message, not when the applications are running.
+     * @param { applications } Array<Identity & { manifestUrl: string }>
```

Comment:
This is backwards, the type goes into the brackets first then then parameter.

Reply:
+1